server/conf/application.conf

```diff
@@ -673,4 +673,20 @@ predicates_multiple_questions_enabled = ${?PREDICATES_MULTIPLE_QUESTIONS_ENABLED
 feature_flag_overrides_enabled = false
 feature_flag_overrides_enabled = ${?FEATURE_FLAG_OVERRIDES_ENABLED}
 
-
+# esri address services
+# address correction
+esri_address_correction_enabled = true
+esri_address_correction_enabled = ${?ESRI_ADDRESS_CORRECTION_ENABLED}
+esri_find_address_candidates_url = "https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/findAddressCandidates"
+esri_find_address_candidates_url = ${?ESRI_FIND_ADDRESS_CANDIDATES_URL}
+# address verification
+esri_address_verification_enabled = true
+esri_address_verification_enabled = ${?ESRI_ADDRESS_VERIFICATION_ENABLED}
+esri_address_verification_labels = ["Seattle"]
+esri_address_verification_labels = ${?ESRI_ADDRESS_VERIFICATION_LABELS}
+esri_address_verification_values = ["Seattle"]
+esri_address_verification_values = ${?ESRI_ADDRESS_VERIFICATION_VALUES}
+esri_address_verification_urls = ["https://gisdata.seattle.gov/server/rest/services/COS/Seattle_City_Limits/MapServer/1/query"]
+esri_address_verification_urls = ${?ESRI_ADDRESS_VERIFICATION_URLS}
+esri_address_verification_paths = ["features[*].attributes.CITYNAME"]
+esri_address_verification_paths =  ${?ESRI_ADDRESS_VERIFICATION_PATHS}
```

Comment:
suggestion (blocking): document these config values and how to set them

Reply:
Would it be better to put the lines setting up Seattle specific values in the application.dev.conf file and I can set up Seattle staging to use the right values? That way we don't leak Seattle stuff into the prod config.